### ISSUE TITLE
Native browser keyboard navigation is prevented in some input fields

### ISSUE THREAD

**Issue by laurens94** [directus/directus#16646]
### Describe the Bug

When an input field is active where the first character cannot be a number (e.g. unique table names), keyboard navigation to switch browser tabs using Ctrl/Cmd + 1, 2, 3 etc. is prevented.

<img width="862" alt="image" src="https://user-images.githubusercontent.com/5780704/204289951-7bf6015d-d8ea-4cbd-a9ef-ef561d8745f5.png">

The input should not be prevented, but instead should be sanitized after changing the value (as right now it's possible to Copy+Paste a number in it anyway).

### To Reproduce

1. Create a collection
2. Activate the empty `Name` field
3. Try using the browsers 'Switch browser tabs' hotkeys (Ctrl/Cmd + 1, 2, 3) to switch tabs
4. It's prevented 😿 

### What version of Directus are you using?

9.21.0

### What version of Node.js are you using?

-

### What database are you using?

-

### What browser are you using?

Firefox

### How are you deploying Directus?

-
